### DISCUSSION TITLE
How to replace content inside `${title}` ?

### DISCUSSION THREAD

**Discussion by azzamsa** [gdh1995/vimium-c#1267] [Q&A]
Hi. 👋🏽 

I am using `map ym copyWindowInfo type="tab" format="[${title}](${url})"` this mapping hundreds of time a day. Copying Webpage title and URL as markdown. But sometimes the page title contains a bracket which breaks the markdown format.

Is there any way to replace the `[]` with `()` in the mapping? 
Maybe something like:

```js
format="[${title}.replace('['.'('](${url})"
```

Thanks!

**Comment by 5HT2** [comment]
Bump, is it possible to do this? Can we somehow escape / encode the sequence?
